crates/index/src/main.rs

```diff
@@ -107,7 +107,7 @@ pub async fn index(index_url: Url) -> Result<(), miette::Error> {
 
         // Continue if there was an error in downloading and skip for now :)
         let metadata = if let Some(metadata) = metadata {
-            metadata.1
+            metadata.unwrap().1
```

Comment:
tbh reading this is a bit confusing :D Is there a way to make this a bit more clear?

Reply:
And maybe if this is a safe unwrap, you could add a little comment? I think Bas would ask for an expect either way :D

Reply:
Yeah so I'm a bit unsure about this index program file as it's kinda half-baked, would rather pull it out of this project entirely and make it kinda nice. Do you still want met to change it?